jax_distribution/systems/ippo_vmap_only.py

```diff
@@ -1,442 +0,0 @@
-# python3
```

Comment:
Should we still include this example? It is not anakin based?

Reply:
I don't think we need to include it, @RuanJohn what do you think?